src/test/java/Homework16.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+@Test
+
+public void registrationNavigation() { {
+
+    ChromeOptions options = new ChromeOptions();
+    options.addArguments("--remote-allow-origins=*");
+    options.addArguments("--start-maximized");
+
+    WebDriver driver = new ChromeDriver(options);
+    driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+    String url = "https://testpro.io/";
```

Comment:
In the homework 16, request Verify that you are redirected to Registration page using Assert method. You need to change url to `https://qa.koel.app/` and then continue verify click registration link and verify.